funcbench/main.go

```diff
@@ -0,0 +1,281 @@
+// Copyright 2019 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package main
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/google/go-github/github"
+	"golang.org/x/oauth2"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+func main() {
+	app := kingpin.New(filepath.Base(os.Args[0]), "benchmark result posting and formating tool.\n-i location of github hook file (even.json)")
+	app.HelpFlag.Short('h')
+	input := app.Flag("input", "path to event.json").Short('i').Default("/github/workflow/event.json").String()
+	kingpin.MustParse(app.Parse(os.Args[1:]))
+
+	data, err := ioutil.ReadFile(*input)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	// Temporary fix for the new Github actions time format, as go-github is not capable of parsing it. This makes the time stamps unusable.
+	txt := string(data)
+	reg := regexp.MustCompile("(.*)\"[0-9]+/[0-9]+/2019 [0-9]+:[0-9]+:[0-9]+ [AP]M(.*)")
+	txt = reg.ReplaceAllString(txt, "$1\"2019-06-11T09:26:28Z$2")
+	data = []byte(txt)
+	// End of the temporary fix.
+
+	event, err := github.ParseWebHook("issue_comment", data)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	switch e := event.(type) {
+	case *github.IssueCommentEvent:
+
+		//Variable setup.
+		branch := readFile("/github/home/ARG_0")
+		if branch == "" {
+			branch = "master"
+		}
+		issueComment := readFile("/github/home/ARG_1")
+		race := readFile("/github/home/ARG_2")
```

Comment:
I think you can have this as 

```suggestion
		race := readFile("/github/home/ARG_2") != "-no-race"
```

this way race will be a boolean